converter/internal/staticconvert/testdata/integrations.yaml

```diff
@@ -159,6 +159,7 @@ integrations:
   redis_exporter:
     enabled: true
     redis_addr: "redis-2:6379"
+    export_key_values: true
```

Comment:
If you want to see this show in `integrations.river` set this to false so it isn't the default value.

Reply:
Good point, updated!